Makefile

```diff
@@ -86,6 +87,14 @@ min_debug:				#: Minimal build with debugging symbols
 	$(MAKE) cmake BUILD_DIR=debug BUILD_TYPE=debug EXTRA_CMAKE_FLAGS=-DVELOX_BUILD_MINIMAL=ON
 	$(MAKE) build BUILD_DIR=debug
 
+benchmarks-build:
+	$(MAKE) release EXTRA_CMAKE_FLAGS="-DVELOX_BUILD_BENCHMARKS=ON"
+
+benchmarks-dump:
+	$(MAKE) benchmarks-build
+	mkdir -p $(BENCHMARKS_DIR)/dumps
+	find $(BENCHMARKS_DIR) -type f -executable -exec {} --bm_min_usec 100000 \;
```

Comment:
next, I'll add a small script to run all these benchmarks and dump results in the dir above.